installer/driverInstall.js

```diff
@@ -113,14 +113,20 @@ var install_node_ibm_db = function(file_url) {
     var clidriverFound = true;
 
     if (process.env.IBM_DB_HOME) {
-      if (fs.existsSync(process.env.IBM_DB_HOME)) {
+      if (platform == 'os390'){
         IBM_DB_HOME = process.env.IBM_DB_HOME;
         IS_ENVIRONMENT_VAR = true;
-      } else {
-      printMsg(process.env.IBM_DB_HOME + " directory does not exist. Please " +
-            "check if you have set the IBM_DB_HOME environment" +
-            " variable\'s value correctly.\n");
-      clidriverFound = false;
+	clidriverFound = true;
+      }else {
+        if (fs.existsSync(process.env.IBM_DB_HOME)) {
+          IBM_DB_HOME = process.env.IBM_DB_HOME;
+          IS_ENVIRONMENT_VAR = true;
+        } else {
+          printMsg(process.env.IBM_DB_HOME + " directory does not exist. Please " +
+                "check if you have set the IBM_DB_HOME environment" +
+                " variable\'s value correctly.\n");
```

Comment:
May want to add the blank space after environment on the second line of printMsg, and removing the prefix space here, to be consistent with the first line.